whirlpool/dryer.py

```diff
@@ -0,0 +1,372 @@
+ATTRVAL_MACHINE_STATE_SYSTEM_INIT = "18"
+ATTRVAL_MACHINE_STATE_CANCELLED = "19"
+
+# Cavity
+ATTR_DOOR_OPEN = "Cavity_OpStatusDoorOpen"
+ATTR_EST_TIME_REMAINING = "Cavity_TimeStatusEstTimeRemaining"
+ATTR_DRUM_LIGHT_ON = "Cavity_DisplaySetDrumLightOn"
+
+# DryCavity
+ATTR_STATUS_EXTRA_POWER_CHANGEABLE = "Cavity_ChangeStatusExtraPowerChangeable"
+ATTR_STATUS_STEAM_CHANGEABLE = "Cavity_ChangeStatusSteamChangeable"
+ATTR_STATUS_CYCLE_SELECT = "DryCavity_ChangeStatusCycleSelect"
+ATTR_STATUS_DRYNESS = "DryCavity_ChangeStatusDryness"
+ATTR_STATUS_MANUAL_DRY_TIME = "DryCavity_ChangeStatusManualDryTime"
+ATTR_STATUS_STATIC_GUARD = "DryCavity_ChangeStatusStaticGuard"
+ATTR_STATUS_TEMPERATURE = "DryCavity_ChangeStatusTemperature"
+ATTR_STATUS_WRINKLE_SHIELD = "DryCavity_ChangeStatusWrinkleShield"
+
+ATTR_CYCLE_SELECT = "DryCavity_CycleSetCycleSelect"
+
+ATTRVAL_CYCLE_SELECT_REGULAR = "1"
+ATTRVAL_CYCLE_SELECT_HEAVY_DUTY = "2"
+ATTRVAL_CYCLE_SELECT_DENIM = "3"
+ATTRVAL_CYCLE_SELECT_DELICATES = "4"
+ATTRVAL_CYCLE_SELECT_WRINKLE_CONTROL = "5"
+ATTRVAL_CYCLE_SELECT_BULKY_ITEMS = "6"
+ATTRVAL_CYCLE_SELECT_QUICK_DRY = "7"
+ATTRVAL_CYCLE_SELECT_SANITIZE = "9"
+ATTRVAL_CYCLE_SELECT_STEAM_REFRESH = "10"
+ATTRVAL_CYCLE_SELECT_TIMED_DRY = "11"
+ATTRVAL_CYCLE_SELECT_COLORS_BRIGHTS = "13"
+ATTRVAL_CYCLE_SELECT_TOWELS = "15"
+ATTRVAL_CYCLE_SELECT_WHITES = "16"
+ATTRVAL_CYCLE_SELECT_NORMAL = "41"
+
+ATTR_DRYNESS = "DryCavity_CycleSetDryness"
+
+ATTRVAL_DRYNESS_LOW = "0"
+ATTRVAL_DRYNESS_LESS = "1"
+ATTRVAL_DRYNESS_NORMAL = "4"
+ATTRVAL_DRYNESS_MORE = "7"
+ATTRVAL_DRYNESS_HIGH = "10"
+
+ATTR_MANUAL_DRY_TIME = "DryCavity_CycleSetManualDryTime"
+
+ATTR_TEMPERATURE = "DryCavity_CycleSetTemperature"
+
+ATTRVAL_TEMPERATURE_AIR = "0"
+ATTRVAL_TEMPERATURE_COOL = "2"
+ATTRVAL_TEMPERATURE_WARM = "5"
+ATTRVAL_TEMPERATURE_WARM_HIGH = "6"
+ATTRVAL_TEMPERATURE_HOT = "8"
+
+ATTR_WRINKLE_SHIELD = "DryCavity_CycleSetWrinkleShield"
+
+ATTRVAL_WRINKLE_SHIELD_OFF = "0"
+ATTRVAL_WRINKLE_SHIELD_ON = "1"
+ATTRVAL_WRINKLE_SHIELD_ON_WITH_STEAM = "2"
+
+ATTR_CYCLE_STATUS_AIR_FLOW_STATUS = "DryCavity_CycleStatusAirFlowStatus"
+ATTR_CYCLE_STATUS_COOL_DOWN = "DryCavity_CycleStatusCoolDown"
+ATTR_CYCLE_STATUS_DAMP = "DryCavity_CycleStatusDamp"
+ATTR_CYCLE_STATUS_DRYING = "DryCavity_CycleStatusDrying"
+ATTR_CYCLE_STATUS_LIMITED_CYCLE = "DryCavity_CycleStatusLimitedCycle"
+ATTR_CYCLE_STATUS_SENSING = "DryCavity_CycleStatusSensing"
+ATTR_CYCLE_STATUS_STATIC_REDUCE = "DryCavity_CycleStatusStaticReduce"
+ATTR_CYCLE_STATUS_STEAMING = "DryCavity_CycleStatusSteaming"
+ATTR_CYCLE_STATUS_WET = "DryCavity_CycleStatusWet"
+
+# DrySys
+ATTR_DAMP_NOTIFICATION_TONE_VOLUME = "DrySys_OpSetDampNotificationToneVolume" 
+
+# Sys
+ATTR_ALERT_TONE_VOLUME = "Sys_OpSetAlertToneVolume"
+
+# XCat
+ATTR_CYCLE_COUNT = "XCat_OdometerStatusCycleCount"
+
+
+class CycleSelect(Enum):
+    Regular = 1
+    HeavyDuty = 2
+    Denim = 3
+    Delicates = 4
+    WrinkleControl = 5
+    Bulky_Items = 6
```

Comment:
```suggestion
    BulkyItems = 6
```
lets make all the names in enums follow `PascalCase`